Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1537,9 +1537,7 @@ def create_search_with_retry(client: Client, fetch_mode: str, offense: Dict, eve
     offense_id = offense['id']
     while num_of_failures <= max_retries:
         try:
-            ret_value = create_events_search(client, fetch_mode, event_columns, events_limit, offense_id, offense['start_time'])
-            time.sleep(EVENTS_MODIFIED_SECS)
-            return ret_value
+            return create_events_search(client, fetch_mode, event_columns, events_limit, offense_id, offense['start_time'])
```

Comment:
From what I could tell, `create_events_search` doesn't raise an exception,
so you'll only iterate once?